@xen-orchestra/build-template/src/index.spec.js

```diff
@@ -0,0 +1,11 @@
+/* eslint-env jest */
+import buildTemplate from '.'
+
+it('builds a template', () => {
```

Comment:
You're testing more than that.

Reply:
Can you please suggest a test name?

Reply:
`'correctly replaces the template's variables'`?